contracts/payloads/IGP101/PayloadIGP101.sol

```diff
@@ -0,0 +1,547 @@
+            address PAXG_XAUT__USDT_VAULT = getVaultAddress(123);
+            address PAXG_XAUT__GHO_VAULT = getVaultAddress(124);
+
+            uint256 CF = 75 * 1e2;
+            uint256 LT = 80 * 1e2;
+            uint256 LML = 83 * 1e2;
+
+            IFluidVaultT1(PAXG_XAUT__USDC_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(PAXG_XAUT__USDC_VAULT).updateLiquidationThreshold(LT);
+            IFluidVaultT1(PAXG_XAUT__USDC_VAULT).updateLiquidationMaxLimit(LML);
+
+            IFluidVaultT1(PAXG_XAUT__USDT_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(PAXG_XAUT__USDT_VAULT).updateLiquidationThreshold(LT);
+            IFluidVaultT1(PAXG_XAUT__USDT_VAULT).updateLiquidationMaxLimit(LML);
+
+            IFluidVaultT1(PAXG_XAUT__GHO_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(PAXG_XAUT__GHO_VAULT).updateLiquidationThreshold(LT);
+            IFluidVaultT1(PAXG_XAUT__GHO_VAULT).updateLiquidationMaxLimit(LML);
+        }
+
+        {
+            address PAXG_XAUT_DEX = getDexAddress(32);
+            {
+                IFluidDex(PAXG_XAUT_DEX).updateMaxSupplyShares(
+                    725 * 1e18 // $5M
+                );
+            }
+        }
+    }
+
+    // @notice Action 6: Set global authorization for DEX Factory
+    function action6() internal isActionSkippable(6) {
+        address global_auth_address = 0xE3e18c563d11ced9B0c9cb8dD0284CF4442bC06a;
+        DEX_FACTORY.setGlobalAuth(global_auth_address, true);
+    }
+
+    // @notice Action 7: Set Launch Limits for GHO-USDe T4 Vault
+    function action7() internal isActionSkippable(7) {
+        {
+            address GHO_USDe_DEX = getDexAddress(37);
+            // GHO-USDe DEX
+            DexConfig memory DEX_GHO_USDe = DexConfig({
+                dex: GHO_USDe_DEX,
+                tokenA: GHO_ADDRESS,
+                tokenB: USDe_ADDRESS,
+                smartCollateral: true,
+                smartDebt: false,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
+                baseBorrowLimitInUSD: 0, // $0
+                maxBorrowLimitInUSD: 0 // $0
+            });
+            setDexLimits(DEX_GHO_USDe); // Smart Collateral
+
+            DEX_FACTORY.setDexAuth(GHO_USDe_DEX, TEAM_MULTISIG, true);
+        }
+        {
+            address GHO_USDe__GHO_USDC_VAULT = getVaultAddress(139);
+            address GHO_USDC_DEX = getDexAddress(4);
+
+            {
+                // Update GHO-USDe<>GHO-USDC vault borrow shares limit
+                IFluidAdminDex.UserBorrowConfig[]
+                    memory config_ = new IFluidAdminDex.UserBorrowConfig[](1);
+                config_[0] = IFluidAdminDex.UserBorrowConfig({
+                    user: GHO_USDe__GHO_USDC_VAULT,
+                    expandPercent: 30 * 1e2, // 20%
+                    expandDuration: 6 hours, // 12 hours
+                    baseDebtCeiling: 4_000 * 1e18, // 4k shares ($8k)
```

Comment:
Base withdrawal limit: $5 Million
Max DEX Supplu shares: $18M (In ratio for Debt Side)
Base Borrow Limit: $5M
Max Borrow Limit: $10M (Capped at DEX)